app/src/main/java/tech/okcredit/appstartupinstrumentation/Application.kt

```diff
@@ -0,0 +1,16 @@
+package tech.okcredit.appstartupinstrumentation
+
+import android.app.Application
+import android.util.Log
+import tech.okcredit.startup_instrumentation.AppStartUpTrace
+
+class Application: Application() {
+    override fun onCreate() {
+        AppStartUpTrace.start()
+        super.onCreate()
+
+        AppStartUpTrace.stop(this) { appStartUpMetrics->
```

Comment:
Should this be at the end of the onCreate method? If yes, can we leave a comment

Reply:
Done